### PR TITLE
[Doc] Update model overview in readme.md

### PR DESCRIPTION
**Pull request by michaelowenliu**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,60 +36,213 @@ Welcome to PaddleSeg! PaddleSeg is an end-to-end image segmentation development
...
+* <img src="./docs/images/f2.png" width="20"/> **Modular Design**: PaddleSeg supports 40+ mainstream *segmentation networks*, developers can start based on actual application scenarios and assemble diversified training configurations combined with modular design of *data enhancement strategies*, *backbone networks*, *loss functions* and other different components to meet different performance and accuracy requirements.
 
 * <img src="./docs/images/f3.png" width="20"/> **High Efficiency**: PaddleSeg provides multi-process asynchronous I/O, multi-card parallel training, evaluation, and other acceleration strategies, combined with the memory optimization function of the PaddlePaddle, which can greatly reduce the training overhead of the segmentation model, all this allowing developers to lower cost and more efficiently train image segmentation model.
 
-## Technical Communication <img src="./docs/images/chat.png" width="30"/>
```
Comment (shiyutang) [inline README.md:47]: This pic is a little big

File: README_CN.md [hunk 2]
```diff
@@ -56,34 +51,197 @@ PaddleSeg是基于飞桨[PaddlePaddle](https://www.paddlepaddle.org.cn)开发的
 ## 技术交流 <img src="./docs/images/chat.png" width="30"/>
```
Comment (shiyutang) [inline README_CN.md:51]: if you need to change, change here as well

File: README_CN.md [hunk 3]
```diff
@@ -125,26 +278,36 @@ PaddleSeg是基于飞桨[PaddlePaddle](https://www.paddlepaddle.org.cn)开发的
...
+* [图像分割套件PaddleSeg全面解析系列](https://blog.csdn.net/txyugood/article/details/111029854)
+* [PaddleSeg学习笔记: 人像分割 HumanSeg](https://blog.csdn.net/libo1004/article/details/118809026)
+
+## 许可证书
+本项目的发布受Apache 2.0 license许可认证。
+
+## 社区贡献
 
 - 非常感谢[jm12138](https://github.com/jm12138)贡献U<sup>2</sup>-Net模型。
 - 非常感谢[zjhellofss](https://github.com/zjhellofss)（傅莘莘）贡献Attention U-Net模型，和Dice loss损失函数。
```
Comment (shiyutang) [inline README_CN.md:313]: change line 319 "- 非常感谢[Herman-Hu-saber](https://github.com/Herman-Hu-saber)(胡慧明)参与贡献 ESPNet"
to "- 非常感谢[Herman-Hu-saber](https://github.com/Herman-Hu-saber)(胡慧明)参与贡献 ESPNetV2"

File: README.md [hunk 4]
```diff
@@ -106,73 +259,49 @@ See [Model Zoo Overview](./docs/model_zoo_overview.md) for more infomation.
...
-* [Domain Adaptation](./contrib/DomainAdaptation)
+## License
 
-## Feedbacks and Contact
-* The dynamic version is still under development, if you find any issue or have an idea on new features, please don't hesitate to contact us via [GitHub Issues](https://github.com/PaddlePaddle/PaddleSeg/issues).
-* PaddleSeg User Group (QQ): 1004738029 or 850378321 or 793114768
+PaddleSeg is released under the [Apache 2.0 license](LICENSE).
 
 ## Acknowledgement
 * Thanks [jm12138](https://github.com/jm12138) for contributing U<sup>2</sup>-Net.
```
Comment (shiyutang) [inline README.md:307]: change 314 as well "ESPNet" to "ESPNetV2"

File: docs/install_cn.md [hunk 5]
```diff
@@ -39,20 +38,24 @@ python -c "import paddle; print(paddle.__version__)"
...
 ```
 cd PaddleSeg
 pip install -r requirements.txt
 
 #如果安装时出现版本错误，可以尝试删除旧版本，重新运行该脚本。
 ```
-## 4.确认环境安装成功
+## 5.确认环境安装成功
 
 执行下面命令，并在PaddleSeg/output文件夹中出现预测结果，则证明安装成功
```
Comment (shiyutang) [inline docs/install_cn.md:61]: English version and Chinese version is not the same.

**Review by shiyutang — CHANGES_REQUESTED** [review]

File: docs/install_cn.md (hunk 5 above)
Comment (michaelowenliu) [reply docs/install_cn.md:61]: done

File: README.md (hunk 4 above)
Comment (michaelowenliu) [reply README.md:307]: done

File: README_CN.md (hunk 3 above)
Comment (michaelowenliu) [reply README_CN.md:313]: done

**Review by shiyutang — APPROVED** [review]
LGTM

**Pull request merged**